recipes/zmqpp/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "zmqpp" %}
+{% set version = "4.2.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zeromq/zmqpp/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c1d4587df3562f73849d9e5f8c932ca7dcfc7d8bec31f62d7f35073ef81f4d29
+
+build:
+  number: 0
+  skip: True  # [win]
```

This is primarily provided a shared library right? If so you should use run_exports: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=run_exports#export-runtime-requirements

```suggestion
  skip: True  # [win]
  run_exports:
    - {{ pin_subpackage('zmqpp', max_pin='x.x') }}
```